video/decode/vd_lavc.c

```diff
@@ -130,6 +130,7 @@ const struct m_sub_options vd_lavc_conf = {
         {0}
     },
     .size = sizeof(struct vd_lavc_params),
+    .change_flags = UPDATE_VD,
```

Comment:
You can just stick this on `vd-lavc-threads`. We don't want this to apply to all the options in here.

Reply:
It depends. `vd-lavc-software-fallback` only needs `UPDATE_HWDEC`. We can always iterate on it later.

Reply:
I've now changed it to be `vd-lavc-threads` only. Should I add it to all the `vd-` options or is there some way to apply it to all of them at the same time? (other then putting hwdec into it's own option config)

Reply:
You don't have to worry about it imo if you don't want to. I'm not exactly sure what options would need what.

Reply:
Then I'll leave it as is and someone else can worry about it if they want to (or me if the need ever arises).